src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx

```diff
@@ -99,28 +145,58 @@ export const ManageDirectQueryDataConnectionsTable: React.FC<ManageDirectQueryDa
       });
   }, [http, notifications.toasts, selectedDataSourceId, featureFlagStatus]);
 
+  const deleteDataSources = useCallback(
+    (connectionName: string | undefined) => {
+      if (!connectionName) return;
+
+      const endpoint =
+        featureFlagStatus && selectedDataSourceId !== undefined
```

Comment:
Nit: seems you're using the name `featureFlagStatus` as is, better to give it a specific name so that it's more readable.